source/mixins/versionable.js

```diff
@@ -0,0 +1,19 @@
+Space.messaging.Versionable = {
+
+  schemaVersion: 1,
+
+  onConstruction(data) {
+    if (_.isObject(data)) this._migrateToLatestVersion(data);
```

Comment:
I know that this is not external API but maybe we should rethink the name of this method `_migrateToLatestVersion`. It is suggesting that migration will always happen.